dhctl/pkg/system/ssh/frontend/upload-script.go

```diff
@@ -106,6 +107,13 @@ func (u *UploadScript) Execute() (stdout []byte, err error) {
 
 	err = scriptCmd.Run()
 	if err != nil {
+		var exitErr *exec.ExitError
+		if errors.As(err, &exitErr) {
+			// Stderr holds a subset of the standard error output from the Cmd.Output method
+			// but we do not call this method
+			exitErr.Stderr = cmd.StderrBytes()
```

Comment:
Can you explain more clearly why this is required exactly? This comment does not provide enough context to me.

Reply:
Added a more detailed comment.